pkg/rtc/participant.go

```diff
@@ -1257,6 +1260,8 @@ func (p *ParticipantImpl) downTracksRTCPWorker() {
 			pkts = pkts[:0]
 			batchSize = 0
 		}
+
+		time.Sleep(5 * time.Second)
```

Comment:
send RTCP immediately after peerconnection connected, is it intentional?

Reply:
nvm, have not see your comment while review

Reply:
Good question @cnderrauber . I think we need to do some more work here. Send SDES immediately. Basically, first iteration through the worker, send only SDES. Then we need to actually adjust frequency based on at least track type (ideally, we should be calculating traffic and ensure that RTCP is not more than 5% of total traffic, but that is more challenging). We should at least change it so that we send RTCP every second for video and every 5 seconds for audio. Will look at that in a subsequent PR.